docker/tutorials/Dockerfile

```diff
@@ -30,3 +30,8 @@ RUN conda install -y -c conda-forge radical.entk
 # RP-Parsl Integration usecase nwchem
 
 RUN conda install -c conda-forge nwchem
+
+# Needed for MPI example in PSI/J
+
+RUN conda install -y -c conda-forge gxx_linux-64
+RUN conda install -y -c conda-forge openmpi
```

Comment:
btw in PR #162 I switched from `conda` to `mamba` (aka advanced `conda`), which is supported by base jupyter docker container, and which helped with parsl-env setup - thus, kinda, either update it here, or later I can update it in that PR (to have it consistent).

and maybe to have this under one `RUN` (== within one layer)?
```
RUN mamba install -y -c conda-forge \
    gxx_linux-64 \
    openmpi
```

Reply:
It looks weird to have two conda install commands and one mamba install command.

But your call. I can make the change here if you think that's better.

Reply:
yeah, would be good to have it here, thanks!

Reply:
K. I'm doing a sanity test now.

Reply:
Seems to work.

Reply:
Great! ready to go